unicodetools/data/security/14.0.0/data/xidAllowed.txt

```diff
@@ -1,5 +1,5 @@
 ﻿# xidAllowed.txt
-# Date: 2021-07-10, 00:52:52 GMT
+# Date: 2021-08-12, 01:13:34 GMT
```

Comment:
Might be good to have some way to exclude date-stamp-only changes: identify such changes locally and revert the file.

Whatever is writing the file could detect this and avoid rewriting (touching) the file.

Reply:
Good idea --> issue #111